Content.Client/GameObjects/EntitySystems/ExamineSystem.cs

```diff
@@ -66,15 +71,35 @@ private bool HandleExamine(ICommonSession session, EntityCoordinates coords, Ent
             return true;
         }
 
+        public override void Update(float frameTime)
```

Comment:
@metalgearsloth Should I change this from Update to FrameUpdate?

Reply:
Yes